src/SFA.DAS.AssessorService.Web.UnitTests/StandardControllerTests/OptOutStandardTests.cs

```diff
@@ -198,15 +199,17 @@ public void OptPOutStandardVersion_ThrowsArgumentException_WhenPostCalledWithNul
         }
 
         [Test]
-        public void OptOutStandardVersion_ThrowsNotFoundException_WhenPostCalledWithExistingVersionForStandard()
+        public async Task OptOutStandardVersion_ReturnsView_WhenPostCalledWithExistingVersionForStandard()
```

Comment:
There should be a redirect test when the view model is invalid

Reply:
Changed